extensions/wikia/PhalanxII/fallback/Phalanx.class.php

```diff
@@ -135,14 +135,14 @@ static public function getFromFilter( $moduleId, $lang = null, $master = false,
 					'reason' => $row->p_reason,
 					'lang' => $row->p_lang
 				);
-				if (!is_null($row->p_expire) && $closestTimestamp > $row->p_expire || !$closestTimestamp) {
+				if ( !is_null( $row->p_expire ) && $closestTimestamp > $row->p_expire || !$closestTimestamp ) {
 					$closestTimestamp = $row->p_expire;
 				}
 			}
 
 			$blocksData['blocks'] = $blocks;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "blocks" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)